gradle/versions.gradle

```diff
@@ -111,6 +111,7 @@ dependencyManagement {
     }
 
     dependency 'org.mockito:mockito-core:3.3.3'
+    dependency 'org.mockito:mockito-junit-jupiter:3.3.3'
```

Comment:
We've been consistently not using this in Teku.  Just using the static methods on `Mockito` rather than the annotations.